### PR TITLE
Adds hugepages config guide to explain defaults and config settings

### LINKED ISSUE #1005: Document how to permanently enable hugepages
This is a placeholder for myself and @bktan8 to discuss where the appropriate place is to document how to enable hugepages. The [DPDK ](https://www.reddit.com/r/ClearLinux/comments/egaof2/how_to_enable_huge_pages_on_clear_linux/)page describe how to do it temporarily but for permanent use you need to do the following. This may need to be documented in any pages describing different `clr-boot-manager` options and would also make sense in any Kubernetes enabling pages. 
```
cat << EOF | sudo tee -a /etc/kernel/cmdline.d/hugepages.conf
default_hugepagesz=2M
hugepagesz=2M
hugepages=1024
EOF
sudo clr-boot-manager update
```

You can verify with `grep ^Huge /proc/meminfo`

### PR DESCRIPTION
**Pull request by eadamsintel**
Closes #1005

Signed-off-by: Adams, Eric <eric.adams@intel.com>

### CODE REVIEW

File: source/guides/maintenance/configure-hugepages.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      ShmemHugePages:        0 kB
+      FileHugePages:         0 kB
+      HugePages_Total:       0
+      HugePages_Free:        0
+      HugePages_Rsvd:        0
+      HugePages_Surp:        0
+      Hugepagesize:       2048 kB
+      Hugetlb:               0 kB
+
+#. If you need 1GB (1,048,576 bytes) of hugepages enabled, then 512 hugepages
```
Comment (mvincerx) [inline source/guides/maintenance/configure-hugepages.rst]: [...] then `FileHugePages` should be set to 512.

File: source/guides/maintenance/configure-hugepages.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+   are needed. Enable it temporarily using the following.
+
+   .. code-block:: bash
+
+      echo 512 | sudo tee /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
```
Comment (mvincerx) [inline source/guides/maintenance/configure-hugepages.rst]: Add: To view the change, run the command in previous step again.

File: source/guides/maintenance/configure-hugepages.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
+.. _enable-huge-pages:
+
+Enable Huge Pages
+########################
+
+In |CL| hugepages are enabled by default for the 2MB. The total number of
```
Comment (ahkok) [inline source/guides/maintenance/configure-hugepages.rst]: `In |CL| hugepages are enabled by default. The default hugepage size is 2MB.`

**Comment by mvincerx** [comment]
@eadamsintel.  Is it possible to make requested revisions today and push? I can help you, if you need.

**Comment by mvincerx** [comment]
LGTM

**Pull request merged**